designsafe/apps/djangoRT/views.py

```diff
@@ -92,7 +92,7 @@ def ticketcreate(request):
             )
 
             ticket = rtModels.Ticket(subject=form.cleaned_data['subject'],
-                                     problem_description=ticket_body,
+                                     problem_description="\n  ".join(ticket_body),
```

Comment:
Is `ticket_body` a string or an array here?
If it's a string isn't this going to add a new line in between each char?

Reply:
Yup, you're right.  I'll work on fixing it.

Reply:
Added a `splitline()` call which resolves this.